.aws/src/main.ts

```diff
@@ -51,15 +51,48 @@ class SnowplowSharedConsumerStack extends TerraformStack {
 
     const pagerDuty = this.createPagerDuty();
 
-    // Create Lambda to consume from pocket-shared-event-bus and send it to snowplow
+    // Create Lambda to consume from pocket-shared-event-bus.
+    //this will forward the event-bridge content to ECS.
+    //if we are not successfully sending to ECS, the messages will be stored in DLQ.
     const sqsEventLambda = new SQSConsumerLambda(this, 'SharedEventConsumer', {
       vpc: pocketVPC,
       pagerDuty,
     });
 
+    //dlq for sqs-sns subscription
+    const snsTopicDlq = new sqs.SqsQueue(this, 'sns-topic-dql', {
```

Comment:
small typo `sns-topic-dql` should be `sns-topic-dlq`

Reply:
✅